soh/soh/GameMenuBar.cpp

```diff
@@ -235,6 +235,8 @@ namespace GameMenuBar {
         // Fairy Revive Percent Restore
         CVar_SetS32("gFairyRevivePercentRestore", 0);
 
+        // Customize Fishing
+        CVar_SetS32("gCustomizeFishing", 0);
         // Instant Fishing
```

Comment:
This CVar also needs to be enabled in the presets that change any of the fishing values as well.

Reply:
I added the randomizer preset for this value in [b91e98d](https://github.com/HarbourMasters/Shipwright/pull/1990/commits/b91e98def6ac401e6f9777ae941d896bb080e4dc) and set it to 1.